test/test_ssl.rb

```diff
@@ -186,8 +186,107 @@ def test_add_certificate_multiple_certs
   end
 
   def test_add_certificate_chain_file
-    ctx = OpenSSL::SSL::SSLContext.new
-    assert ctx.add_certificate_chain_file(Fixtures.file_path("chain", "server.crt"))
```

Comment:
Maybe create a separate test, since this is still a useful test, no?

Reply:
You commented about `assert ctx.add_certificate_chain_file`?
Now, `SSLContext#add_certificate_chain_file` returns self, not boolean.
I modify tests to call `assert_nothing_raised`.

Reply:
I modify `add_certificate_chain_file` to return self. It is the reason the following as:

1. It is exceptional situations to fail to add the certificate or the private key. So, when failed to add, `add_certificate_chain_file` be better to raise an error instead of returning false, I think.
2. `add_certificate`(method for a similar purpose) raises an error if failed to add, too.